shaping/shaper.go

```diff
@@ -94,8 +94,18 @@ func (t *HarfbuzzShaper) Shape(input Input) Output {
 	font.XScale = int32(input.Size.Ceil()) << scaleShift
 	font.YScale = font.XScale
 
+	features := make([]harfbuzz.Feature, len(input.FontFeatures))
```

Comment:
Can we make this a reusable buffer on the shaper implementation rather than a newly-allocated buffer per-shaping?

Reply:
Fair enough !